### PR TITLE
Feature/ccp 253 

### PR DESCRIPTION
**Pull request by kkrish39**
Raising a new PR for the Review Request ticket ( CCP-253 )

### CODE REVIEW

File: Conceptpower+Spring/src/main/java/edu/asu/conceptpower/app/db/DatabaseClient.java [hunk 1]
```diff
@@ -304,5 +304,15 @@ private String getLikePattern(final String searchTerm) {
           }
           
         };
-      }
+    }
+    
+    @Override
+    public List<ConceptEntry> getAllConceptsByStatusPaginated(ReviewStatus status, Integer page, Integer pageSize) {
```
Comment (jdamerow) [inline Conceptpower+Spring/src/main/java/edu/asu/conceptpower/app/db/DatabaseClient.java]: how about `getConceptsByStatus`, the rest is implicit and makes the method name unnecessarily lengthly.

File: Conceptpower+Spring/src/main/java/edu/asu/conceptpower/app/manager/impl/ConceptManager.java [hunk 2]
```diff
@@ -641,4 +642,14 @@ public ConceptEntry getOriginalConceptEntry(String id) {
     public Iterable<ConceptEntry> getAllConcepts() {
         return client.getAllConcepts();
     }
+    
+    @Override
+    public List<ConceptEntry> getAllConceptsByStatusPaginated(ReviewStatus status, Integer page, Integer pageSize) {
```
Comment (jdamerow) [inline Conceptpower+Spring/src/main/java/edu/asu/conceptpower/app/manager/impl/ConceptManager.java]: `getConceptsByStatus`

File: Conceptpower+Spring/src/main/java/edu/asu/conceptpower/web/ReviewRequestController.java [hunk 3]
```diff
@@ -6,45 +6,61 @@
...
-        reviewRequest.setRequester(principal.getName());
-        reviewRequest.setCreatedAt(OffsetDateTime.now());
-        requestsMgr.addReviewRequest(reviewRequest);
-        
-        return reviewRequest;
+    	reviewRequest.setRequester(principal.getName());
+    	reviewRequest.setCreatedAt(OffsetDateTime.now());
+    	requestsMgr.addReviewRequest(reviewRequest);
+    	
+    	return reviewRequest;
```
Comment (jdamerow) [inline Conceptpower+Spring/src/main/java/edu/asu/conceptpower/web/ReviewRequestController.java]: ?

File: Conceptpower+Spring/src/main/java/edu/asu/conceptpower/web/ViewOpenReviewRequestsController.java [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+
+        pageNo = pageNo < 0 ? 0 : pageNo;
+        numRecordsPerPage = numRecordsPerPage < 0 ? defaultNumRecordsPerPage : numRecordsPerPage;
+
+        int totalPageCount = (int) Math.ceil(
+            (double) conceptsManager.getNumberOfConceptsByStatus(ReviewStatus.OPENED) / (double) numRecordsPerPage);
+
+        List < ConceptEntry > openReviewConcepts = conceptsManager.getAllConceptsByStatusPaginated(ReviewStatus.OPENED,
+            pageNo, numRecordsPerPage);
+        List < ConceptEntryWrapper > foundConcepts = new ArrayList < > ();
```
Comment (jdamerow) [inline Conceptpower+Spring/src/main/java/edu/asu/conceptpower/web/ViewOpenReviewRequestsController.java]: let's remove the blanks to the pointy brackets

File: Conceptpower+Spring/src/main/java/edu/asu/conceptpower/web/ViewOpenReviewRequestsController.java [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+
+        try {
+            if (!openReviewConcepts.isEmpty()) {
+                foundConcepts = wrapperCreator
+                    .createWrappers(openReviewConcepts.toArray(new ConceptEntry[openReviewConcepts.size()]));
+            }
+        } catch (LuceneException e) {
+            model.addAttribute(indexerStatus, e.getMessage());
+            e.printStackTrace();
```
Comment (jdamerow) [inline Conceptpower+Spring/src/main/java/edu/asu/conceptpower/web/ViewOpenReviewRequestsController.java]: ?

File: Conceptpower+Spring/src/main/webapp/WEB-INF/views/home.html [hunk 6]
```diff
@@ -6,423 +6,546 @@
...
+	   <input type="hidden" name="conceptId" id="conceptId" value=""/>
+	   
+	   <div th:insert="layouts/modals/conceptdetails :: conceptdetails"></div>
+	   <div th:insert="layouts/modals/requestreview :: addrequest"></div>
+	   <div th:insert="layouts/modals/resolvereview :: resolverequest"></div>
+	   <div th:insert="layouts/modals/reopenreview :: reopenrequest"></div>
+	   <div th:insert="layouts/modals/invalidrequest :: invalidRequest"></div>
+	   <div sec:authorize="!isAuthenticated()" th:insert="layouts/modals/signin :: signin"></div>
+	   <div th:insert="layouts/modals/deleteconcept :: deleteConcept"></div>
+	</body>
```
Comment (jdamerow) [inline Conceptpower+Spring/src/main/webapp/WEB-INF/views/home.html:550]: what has actually been changed in this file? Please don't change parts that don't need to be changed.

**Review by jdamerow — CHANGES_REQUESTED** [review]

File: Conceptpower+Spring/src/main/webapp/WEB-INF/views/home.html (hunk 6 above)
Comment (kkrish39) [reply Conceptpower+Spring/src/main/webapp/WEB-INF/views/home.html:550]: I'm sorry, yeah. I have removed a couple of AJAX functions and put them in a common js file.

Comment (jdamerow) [reply Conceptpower+Spring/src/main/webapp/WEB-INF/views/home.html:550]: please revert formatting changes and reread PR guidelines.

**Pull request closed without merging**